formapp/app.py

```diff
@@ -251,8 +252,10 @@ def newStudent(remote, username, bot=False):
             if username in user:
                 if not teacher:
                     sD.studentDict[remote]['perms'] = sD.settings['perms']['admin']
+                if tags = True
                 else:
                     sD.studentDict[remote]['perms'] = int(user[3])
+                sD.studentDict[remote]['tags'] = json.loads(user[13])
```

Comment:
How does this not cause a huge error?

Reply:
Ive been trying to fix this, it only breaks when making a new user, but i have no idea where else to put this, it needs this to load the tags for users. if i remove this, the tags stop working